installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -6,16 +6,46 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+You can install OpenShift on a single node using an ISO that you generate.
+
 ifndef::openshift-origin[]
-include::modules/install-sno-generating-the-discovery-iso-with-the-assisted-installer.adoc[leveloffset=+1]
-endif::openshift-origin[]
 
-include::modules/install-sno-generating-the-discovery-iso-manually.adoc[leveloffset=+1]
+== Installing single-node OpenShift using the Assisted Installer
 
-include::modules/install-sno-installing-with-usb-media.adoc[leveloffset=+1]
+To install OpenShift on a single node, first generate the installation ISO, then boot the server from the ISO. You can monitor the installation using the assisted installer.
```

Comment:
This kinda sounds like the assisted installer is only there for monitoring, but it's much more than that. Also the term "installation ISO" is not entirely correct - in the context of the assisted installer, it's a discovery ISO.

Perhaps instead this paragraph should explain what the assisted installer is and how it can help you install single node openshift, something along the lines of "To install OpenShift on a single node, it's recommended to use the Assisted Installer, a simple web-based wizard that guides you and manages the installation"